### ISSUE TITLE
--no-verify to bypass is not currently working...

### ISSUE THREAD

**Issue by FJunior225** [typicode/husky#137]
Hello,

I am creating a custom prepare-commit-msg hook that appends an ID to the commit msg. If no ID is found within the message or branch name, I log an error and `exit(1)`. I `exit(1)` because I do not want to permit someone to commit a message without an ID. For some unknown reason, `--no-verify` does not bypass the commit message if an error occurs. I was hoping someone here can assist... 
```
fj/test-prepare ✗
▶ git commit -m "remove test." --no-verify

> husky - npm run -s preparecommitmsg

No JIRA ID found. Skipping prepend.

> husky - prepare-commit-msg hook failed (add --no-verify to bypass)
> husky - to debug, use 'npm run preparecommitmsg'
```
The script within the project I am adding a commit has the following in the `package.json`:
`"preparecommitmsg": "node node_modules/***/src/preparecommitmsg-hook.js"`

**Comment by alexeyten** [comment]
https://git-scm.com/docs/githooks

prepare-commit-msg [...] and it is not suppressed by the --no-verify option.

Actually, you should use `commit-msg` hook to verify commit message

**Comment by FJunior225** [comment]
@alexeyten that makes sense - is it possible to update Husky to not show `husky - prepare-commit-msg hook failed (add --no-verify to bypass)` message for prepare-commit-msg hook? Because if I am reading that correctly, `--no-verify` will have no impact on the hook and showing that message may be misleading.

**Comment by typicode** [comment]
@alexeyten @FJunior225 very good point, I'll add it in the next release 👍 
Thank you :)

**Comment by typicode** [comment]
Fixed in `v0.14`
